### PR TITLE
fix: add consent settings into the values.yaml file

### PR DESCRIPTION
**Pull request by leleueri**
fixes AM-1046

### CODE REVIEW

File: helm/tests/gateway-configmap_test.yaml [hunk 1]
```diff
@@ -318,3 +318,20 @@ tests:
...
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+      - matchRegex:
+          path: data.[gravitee.yml]
+          pattern: "[ ]{2}ip: true"
+      - matchRegex:
+          path: data.[gravitee.yml]
+          pattern: "[ ]{2}user-agent: false"
```
Comment (passionne) [inline helm/tests/gateway-configmap_test.yaml:337]:
```suggestion      - matchRegex:
          path: data.[gravitee.yml]
          pattern: |
            consent:
              ip: true
              user-agent: false
```
This does the job also and seems clearer imo.

File: helm/tests/gateway-configmap_test.yaml [hunk 2]
```diff
@@ -318,3 +318,19 @@ tests:
...
+    asserts:
+      - hasDocuments:
+          count: 1
+      - isKind:
+          of: ConfigMap
+          path: data.[gravitee.yml]
+          pattern: |
+            consent:
+              ip: true
+              user-agent: false
```
Comment (passionne) [inline helm/tests/gateway-configmap_test.yaml:336]:
```suggestion
      - matchRegex:
          path: data.[gravitee.yml]
          pattern: |
            consent:
              ip: true
              user-agent: false
```
Sorry, misclic when CC and GH interface issue... this should be better 😅

**Review by passionne — APPROVED** [review]
BTW, thanks for updating the `CHANGELOG.md` I missed it last time 😅.

**Comment by leleueri** [comment]
skip integration tests as this PR update the HelmChart

**Pull request merged**